public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -541,7 +601,7 @@ block dart-type-exceptions
 :marked
   ### Remember the brackets
   The brackets tell Angular to evaluate the template expression.
-  If we forget the brackets, Angular treats the string as a constant and *initializes the target property* with that string.
+  If directive forget the brackets, Angular treats the string as a constant and *initializes the target property* with that string.
```

Comment:
How about "if you omit the brackets"

I thought I fixed all the "we" to "you". Hmmm